src/sonic-yang-models/yang-models/sonic-types.yang

```diff
@@ -82,11 +86,25 @@ module sonic-types {
         }
     }
 
+    typedef hwsku {
+        type string {
+            length 1..255;
+            /* Should we list all hwsku here */
+        }
+    }
+
     typedef vlan_tagging_mode {
         type enumeration {
             enum tagged;
             enum untagged;
             enum priority_tagged;
```

Comment:
default value

Reply:
tagging_mode is a mandatory field in sonic-vlan.yang, so the user must enter it, i.e. no default needed.